### PR TITLE
Model specs

### PR DESCRIPTION
**Pull request by Abe1able**
### In this milestone of the project, I added validations to the models and created specs for them

These are the details:
Added the following validations:

**For the User model:**
> Name must not be blank.
> PostsCounter must be an integer greater than or equal to zero.

**For the Post model:**
> Title must not be blank.
> Title must not exceed 250 characters.
> CommentsCounter must be an integer greater than or equal to zero.
> LikesCounter must be an integer greater than or equal to zero.

**I added unit specs for all the models' methods and validations.**

### CODE REVIEW

File: spec/models/post_spec.rb [hunk 1]
```diff
@@ -48,4 +48,4 @@
       expect(@post1.five_recent_comments.count).to eql(0)
```
Comment (rloterh) [inline spec/models/post_spec.rb:48]: - [x] Some of the model specs are failing. Let's ensure all model specs are passing.

**Failing specs:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Abel, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🏁 🚦 ⛪ 🚜 🏃 🌴 🏫 🌳 🏘️

## Highlights 👍  
- Model validations properly done!
- ReadMe looks good!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
Kindly also include instructions in your ReadMe on how to run the model spec test cases of your project, under the _Getting Started_ section.
Implementing the above change(s) will make your ReadMe/repository stand out even better! 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/post_spec.rb (hunk 1 above)
Comment (Abe1able) [reply spec/models/post_spec.rb:48]: Fixed, thank you very much.

**Review by codecaiine — APPROVED** [review]
Hi @Abe1able , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-Good test implementation ✔️ 
- All tests are passed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**